guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,701 @@
+semantic segmentation.
+
+Now that our classifier is built, let's apply it to this cute cat picture!
+"""
+
+filepath = tf.keras.utils.get_file(origin="https://i.imgur.com/9i63gLN.jpg")
+image = keras.utils.load_img(filepath)
+image = np.array(image)
+keras_cv.visualization.plot_image_gallery(
+    [image], rows=1, cols=1, value_range=(0, 255), show=True, scale=4
+)
+
+"""
+Next, let's get some predictions from our classifier:
+"""
+
+predictions = classifier.predict(np.expand_dims(image, axis=0))
+"""
+Predictions come in the form of softmax-ed category rankings.
+We can find the index of the top classes using a simple argsort function:
+"""
+top_classes = predictions[0].argsort(axis=-1)
+
+
+"""
+In order to decode the class mappings, we can construct a mapping from
+category indices to ImageNet class names.
+For conveneince, I've stored the ImageNet class mapping in a GitHub gist.
+Let's download and load it now.
+"""
+classes = keras.utils.get_file(
+    origin="https://gist.githubusercontent.com/LukeWood/62eebcd5c5c4a4d0e0b7845780f76d55/raw/fde63e5e4c09e2fa0a3436680f436bdcb8325aac/ImagenetClassnames.json"
+)
+with open(classes, "rb") as f:
+    classes = json.load(f)
+"""
+Now we can simply look up the class names via index:
+"""
+top_two = [classes[str(i)] for i in top_classes[-2:]]
+print("Top two classes are:", top_two)
+
+"""
+Great!  Both of these appear to be correct!
+However, the top class here is "Velvet".
+We're trying to classify Cats VS Dogs.
+We don't care about the velvet blanket!
+
+Ideally, we'd have a classifier that only performs computation to determine if
+an image is a cat or a dog, and has all of its resources dedicated to this task.
+This can be solved by fine tuning our own classifier.
+
+# Fine tuning a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_intermediate.png)
+
+When labeled images specific to our task are available, fine-tuning a custom
+classifier can improve performance.
+If we want to train a Cats vs Dogs Classifier, using explicitly labeled Cat vs
+Dog data should perform better than the generic classifier!
+For many tasks, no relevant pretrained model
+will be available (e.g., categorizing images specific to your application).
+
+First, let's get started by loading some data:
+"""
+
+BATCH_SIZE = 32
+IMAGE_SIZE = (224, 224)
+AUTOTUNE = tf.data.AUTOTUNE
+tfds.disable_progress_bar()
+
+data, dataset_info = tfds.load("cats_vs_dogs", with_info=True, as_supervised=True)
+train_steps_per_epoch = dataset_info.splits["train"].num_examples // BATCH_SIZE
+train_dataset = data["train"]
+
+num_classes = dataset_info.features["label"].num_classes
+
+resizing = keras_cv.layers.Resizing(
+    IMAGE_SIZE[0], IMAGE_SIZE[1], crop_to_aspect_ratio=True
+)
+
+
+def preprocess_inputs(image, label):
+    image = tf.cast(image, tf.float32)
+    return resizing(image), tf.one_hot(label, num_classes)
+
+
+train_dataset = train_dataset.shuffle(10 * BATCH_SIZE).map(
```

Comment:
Would love a comment about why we're shuffling here and why `10 * BATCH_SIZE` is the way to do it